### PR TITLE
add qhull wrapper jll

### PR DESCRIPTION
**Pull request by SimonDanisch**
Tested it locally and seems to work well :) 
I have a question though, due to https://github.com/JuliaPackaging/Yggdrasil/pull/3662, it seems like I need to specify a compat for the `Dependency`, since it considers the old version to be newer than 8?
CC: @stevengj

### CODE REVIEW

**Comment by giordano** [comment]
There is no license

**Comment by SimonDanisch** [comment]
CC: @santiagobadia @victorsndvg @fverdugo, added this as a license https://github.com/JuliaPackaging/Yggdrasil/pull/5588/files#diff-ac28c35c7c79ee8d0984f37972032915c680af76d62ecb3a50bbe270a66bb566 taken from MiniQhull.jl

**Comment by SimonDanisch** [comment]
> Dependency Qhull_jll does not have a mapping for artifact Qhull for platform armv6l-linux-musleabihf-libgfortran3-cxx03

Hm, I guess new platforms were added since Qhull_jll has been build the last time?

**Comment by fverdugo** [comment]
@SimonDanisch Thanks for packaging this!

File: Q/QhullMiniWrapper/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("qhullwrapper", :qhullwrapper)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = Dependency[
+    Dependency(PackageSpec(name="Qhull_jll"), compat="8")
```
Comment (giordano) [inline Q/QhullMiniWrapper/build_tarballs.jl:28]: You have to use a version which actually exists: https://docs.binarybuilder.org/stable/reference/#BinaryBuilderBase.Dependency

> The optional keyword argument `compat` can be used to specify a string for use in the `Project.toml `of the generated Julia package. If `compat` is non-empty and `build_version` is not passed, the latter defaults to the minimum version compatible with the `compat` specifier.

```suggestion
dependencies = [
    Dependency(PackageSpec(name="Qhull_jll"), compat="8.0.1000")
```

**Comment by SimonDanisch** [comment]
Cool cool cool :) Seems like that fixed all problems ;)

**Pull request merged**